src/pybind/mgr/dashboard/constraints.txt

```diff
@@ -3,5 +3,5 @@ more-itertools==4.1.0
 PyJWT==2.0.1
 bcrypt==3.1.4
 python3-saml==1.4.1
-requests==2.25.1
+requests~=2.25
```

Comment:
As we've had these recent failures on this part I though it could be a good idea to relax the pinning and allow a range/soft-pinning instead. That's actually giving `pip` more leeway on which version to install... But given that I've also hard-pinned the `teuthology` repo, this shouldn't cause any new issues, right? I'll replace this with a hard-pinning you're suggesting then.